### ISSUE TITLE
What is the aux_gts in the training code? (Dataloader)

### ISSUE THREAD

**Issue by suesuekkim** [Genie-Kim/PintheMemory#4]
In your meta-learning algorithm in your train-code, What does aux_gts exactly mean?
Is the data gts and aux_gts are the same?

Thank you for providing the code of your work!

**Comment by Genie-Kim** [comment]
Thank you for your interest!

We adopt the auxiliary per-pixel cross-entropy loss proposed in PSPNet [1] with a coefficient of 0.4 to make a fair comparison with the RobustNet [2].
The aux_gts are the same as the original gts in our settings.
However, in RobustNet [2], it seems that aux_gts are different depending on their ablation setting.
We just kept that option as is.

[1] Zhao et al, Pyramid scene parsing network. In CVPR, 2017.
[2] Choi et al, Robustnet: Improving domain generalization in urban-scene segmentation via instance selective whitening. In CVPR, 2021.
